### PR TITLE
Changes for building OCNE node image on kind 0.14.0

### PR DESCRIPTION
**Pull request by fredtibbitts**
Initial set of changes to the upstream v0.14.0 tag to support building and running a node image that uses OCNE. Includes adding the SECURITY.md and THIRD_PARTY_LICENSES.txt files.

### CODE REVIEW

File: images/base/Dockerfile [hunk 1]
```diff
@@ -12,207 +12,94 @@
...
 RUN echo "Adjusting systemd-tmpfiles timer" \
     && sed -i /usr/lib/systemd/system/systemd-tmpfiles-clean.timer -e 's#OnBootSec=.*#OnBootSec=1min#'
 
+RUN mkdir /license
+COPY LICENSE README.md THIRD_PARTY_LICENSES.txt /license/
+
 # squash
-FROM scratch
+#FROM scratch
+FROM ghcr.io/oracle/oraclelinux:8-slim
```
Comment (fredtibbitts) [inline images/base/Dockerfile:91]: I'm not sure about this change, I don't know if there's something we're now doing that requires an OS. It would be nice to be able to use scratch for this. Need to confirm with @theabmitra that this is really necessary.

**Review by michael-cico — APPROVED** [review]

File: images/base/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+LICENSE
```
Comment (desagar) [inline images/base/.gitignore:1]: why are these files needed to be in the .gitignore?

Comment (fredtibbitts) [reply images/base/.gitignore:1]: The files exist in the root of the repository, but the Dockerfile is in images/base, and the Dockerfile can't copy files from directories above where the `docker build` command is run. So the options are 1) copy the files from the root to the directory containing the Dockerfile, or 2) run `docker build` from the root and use `-f` to specify the Dockerfile. Option two required more changes to the Dockerfile so I went with copying the files, thus the `.gitignore`.

**Review by desagar — APPROVED** [review]

File: images/base/.gitignore (hunk 2 above)
Comment (desagar) [reply images/base/.gitignore:1]: thanks for the explanation :)

**Review by desagar — APPROVED** [review]

**Review by michael-cico — APPROVED** [review]

**Review by michael-cico — APPROVED** [review]

**Review by desagar — APPROVED** [review]
